### PR TITLE
Changes for APP Manager v2 usertesting

### PR DESCRIPTION
**Pull request by biyeun**
@orangejenny 

feb-28-2017 02-10-45
feb-28-2017 02-11-49
feb-28-2017 02-12-01
feb-28-2017 02-12-54
feb-28-2017 02-13-32

screen shot 2017-02-28 at 2 16 19 am
screen shot 2017-02-28 at 2 15 49 am
screen shot 2017-02-28 at 2 15 37 am
screen shot 2017-02-28 at 2 15 28 am
screen shot 2017-02-28 at 2 15 16 am

### CODE REVIEW

**Comment by czue** [comment]
pretty :heart_eyes:

**Comment by benrudolph** [comment]
wow! 🌈

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -5246,6 +5246,9 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
     grid_form_menus = StringProperty(default='none',
                                      choices=['none', 'all', 'some'])
 
+    def has_modules(self):
+        return len(self.modules) > 0
```
Comment (orangejenny) [inline corehq/apps/app_manager/models.py]: Might as well have a `not self.is_remote_app()` check here. (I just got bitten by this.)

**Comment by orangejenny** [comment]
Just one non-blocking comment.

Do you know why this PR isn't showing test results?

**Comment by biyeun** [comment]
@orangejenny build passed and responded

**Review by orangejenny — APPROVED** [review]

**Pull request merged**